connect/transforms/src/main/java/org/apache/kafka/connect/transforms/Cast.java

```diff
@@ -154,10 +158,14 @@ private R applyWithSchema(R record) {
         final Struct updatedValue = new Struct(updatedSchema);
         for (Field field : value.schema().fields()) {
             final Object origFieldValue = value.get(field);
+            log.trace("Cast SMT: Original value: {}", origFieldValue);
             final Schema.Type targetType = casts.get(field.name());
             final Object newFieldValue = targetType != null ? castValueToType(field.schema(), origFieldValue, targetType) : origFieldValue;
+            log.trace("Cast SMT: New value: {}", newFieldValue);
```

Comment:
This is a useful log message. But since in a busy Connect worker it's unlikely these log two messages will be adjacent, how about instead using a single log message:

    log.trace("Cast field '{}' from '{}' to '{}'", field.name(), origFieldValue, newFieldValue);